### PR TITLE
Cycling

### PR DESCRIPTION
**Pull request by nesmon**
Add cycling option for go to the first page when reach last and click on next 
or go to last when click on back at first page

### CODE REVIEW

File: lib/PaginationEmbed.js [hunk 1]
```diff
@@ -36,6 +37,7 @@ class PaginationEmbed {
         this.page       = options.startPage     || 1;
         this.maxMatches = options.maxMatches    || 50;
         this.timeout    = options.timeout       || 300000;
+        this.cycling = options.cycling    || false
```
Comment (riyacchi) [inline lib/PaginationEmbed.js]: Indentation is messed up and should be indented with the other properties.

File: lib/PaginationEmbed.js [hunk 2]
```diff
@@ -168,6 +176,7 @@ module.exports = {
         await paginationEmbed.initialize();
         paginationEmbed.run();
 
+
```
Comment (knht) [inline lib/PaginationEmbed.js]: Unnecessary line should be removed
```suggestion
```

File: lib/PaginationEmbed.js (hunk 1 above)
Comment (knht) [inline lib/PaginationEmbed.js]: Indentation is messed up and should be indented with the other properties.

File: lib/PaginationEmbed.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ class PaginationEmbed {
      * @property {Boolean} options.showPageNumbers Whether or not to show the current page index over the embed. Defaults to: true
      * @property {Boolean} options.extendedButtons Whether or not to show extended control buttons besides standard pagination (First & Last page, deleting)
      * @property {Number} options.maxMatches How often the reaction handler should listen for a reaction (How often the paginator can be used). Defaults to: 50. Maximum: 100
+     * @property {Boolean} options.cycling If back to rist page when you click on next in last page or click last page when you click on back in first page. Defaults to: false
```
Comment (knht) [inline lib/PaginationEmbed.js]: The english isn't in line with the rest of the documentation and contains typos.

```suggestion
     * @property {Boolean} options.cycling Cycle through all embeds jumping from the first page to the last page on going back and from the last page to the first page going forth. Defaults to: false
```

File: README.md [hunk 4]
```diff
@@ -23,6 +23,9 @@ EmbedPaginator.createPaginationEmbed(message, embeds, options);
     - **Boolean** `options.extendedButtons` - Whether to show advanced pagination buttons (Jump to First & Last page)
         - Optional: **Yes**
         - Default: **False**
+    - **Boolean** `options.cycling` - If back to rist page when you click on next in last page or click last page when you click on back in first page.
```
Comment (knht) [inline README.md]: Same issue as the JSDoc
```suggestion
    - **Boolean** `options.cycling` - Cycle through all embeds jumping from the first page to the last page on going back and from the last page to the first page going forth.
```

**Review by knht — CHANGES_REQUESTED** [review]

**Comment by nesmon** [comment]
I update the repo with suggest send me

**Review by knht — CHANGES_REQUESTED** [review]
The changes made are fine, but you're missing the previously pointed out indentation in line 40

**Review by knht — APPROVED** [review]

**Pull request merged**